core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -805,8 +808,10 @@ private[spark] class MapOutputTrackerMaster(
     shuffleStatuses.get(shuffleId) match {
       case Some(shuffleStatus) =>
         shuffleStatus.updateMapOutput(mapId, bmAddress)
+      case None if shuffleMigrationEnabled =>
+        logDebug(s"Asked to update map output for unknown shuffle ${shuffleId}")
       case None =>
-        logError(s"Asked to update map output for unknown shuffle ${shuffleId}")
+        logWarn(s"Asked to update map output for unknown shuffle ${shuffleId}")
```

Comment:
Let us not change this.
The discussion above was whether we should make something was was `error` earlier into `debug` or not.
Note - just because migration is enabled does not tell us if this failure is due to migration - it need not be.